app/components/course-page/course-stage-step/community-solutions-list.ts

```diff
@@ -19,9 +22,19 @@ export default class CommunitySolutionsListComponent extends Component<Signature
   rippleSpinnerImage = rippleSpinnerImage;
 
   @service declare authenticator: AuthenticatorService;
+  @service declare coursePageState: CoursePageStateService;
 
   @tracked configureGithubIntegrationModalIsOpen = false;
 
+  get shouldShowStageIncompleteModal() {
+    return (
+      !this.args.stageIncompleteModalWasDismissed &&
+      this.coursePageState.currentStep.globalPosition > 4 &&
```

Comment:
Instead of using `globalPosition`, can we filter by `courseStage.isFirst` & `courseStage.isSecond`? `globalPosition` is a bit indirect - if we add another "setup" step for example, the `globalPosition` required here would change from 4 to 5.